01/app.js

```diff
@@ -0,0 +1,9 @@
+const buttonEl = document.querySelector('button');
+
+if (!buttonEl) {
```

Comment:
Warto zwrócić uwagę mentee dlaczego to działa - wersję skrócone nie zawsze muszą być zrozumiałe :)